presto-native-execution/CMakeLists.txt

```diff
@@ -160,9 +159,18 @@ endif()
 include_directories(SYSTEM ${FOLLY_INCLUDE_DIRS})
 
 # Include third party header files
+
+# Find openssl@1.1 folder.
 find_path(OPT_OPENSSL_DIR NAMES opt/openssl@1.1)
+if(${OPT_OPENSSL_DIR} STREQUAL "OPT_OPENSSL_DIR-NOTFOUND")
```

Comment:
Can we instead make OPT_OPENSSL_DIR an env variable which if not set does this particular search ? 
Also if its not set and we dont find opt/openssl , would it be more useful to just print out a message that it couldnt find opt/openssl@1.1 and leave it to user to explicitly set it for the env ?

Reply:
@kgpai 
We could sort this many ways.
My thoughts:
- I actually like unconditional search because as a user you don't need to specify anything extra.
- The search was there before, I just made sure it finds the folder in my weird environment as well.
- I thought about writing a message if the path not found, but then changed my mind. People seems like just ignore the warnings and the find_package() should error out if not found.
- How can we make OPT_OPENSSL_DIR an env variable? I thought as setting it here is equivalent on declaring such variable for the life of the run.

Reply:
For the last one , you can just check if an env variable named OPT_OPENSSL_DIR is defined and if so use the value.

Reply:
We can do that.
I thought about that too.
Two things stopped me:
- Who would go into trouble defining one, if the build works w/o doing it.
- This is a special knowledge - to know about OPT_OPENSSL_DIR existence, which most of the users won't possess.